relay-server/src/actors/processor.rs

```diff
@@ -2176,6 +2176,12 @@ impl EnvelopeProcessorService {
         let event = &mut state.event;
         let config = &state.project_state.config;
 
+        if config.datascrubbing_settings.scrub_data {
+            if let Some(event) = event.value_mut() {
+                scrub_graphql(event);
+            }
+        }
```

Comment:
So the reason for moving this here is that inside the processor, we do not have access to `datascrubbing_settings`? Might be worth pointing that out in the PR description.